sdk/go/channels_test.go

```diff
@@ -141,7 +141,7 @@ func TestCreateChannels(t *testing.T) {
 		res, err := mainfluxSDK.CreateChannels(tc.channels, tc.token)
```

Comment:
Why changing these other files? Is it lint that is complaining?

Reply:
Looks like yes, so I will approve.

Reply:
yes